multinet/api/models/workspace.py

```diff
@@ -81,6 +89,19 @@ def get_user_permission(self, user: User) -> Optional[WorkspaceRole]:
         """Get the WorkspaceRole for a given user on this workspace."""
         return WorkspaceRole.objects.filter(workspace=self.pk, user=user.pk).first()
 
+    def get_user_permission_string(self, user: User) -> str:
+        """Get a string representation of a user's workspace role."""
```

Comment:
I think a comment here describing the edge cases for the workspace owner and a public workspace reader would be good. Basically, I want to distinguish this from `get_user_permission`, as that doesn't take those things into account, and this function differs from that considerably. I'd ideally like a better way to make this distinction aside from a code comment, but I'm not sure what the best way to do that is.

Reply:
Let me know if you have any thoughts on the new comment that I added.